lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -46,13 +46,7 @@
         </button>
     <% } else {%>
         <div class="no-action-message">
-            <%- gettext('Coming Soon') %>
-        </div>
-        <div class="enrollment-opens">
-            <%- gettext('Enrollment Opens on') %>
-            <span class="enrollment-open-date">
-                <%- enrollment_open_date %>
-            </span>
+            <%- gettext('Not Currently Available') %>
```

Comment:
@rlucioni Do you have some evidence to support the claim that `course enrollment start and end dates are always left blank`? Did you check each and every course in our production system? Basically, I am interested to know how you arrived at that statement. I am especially interested why you used the word `always`